### DISCUSSION TITLE
Width/height/top/left input precision

### DISCUSSION THREAD

**Discussion by badiuciprian** [GrapesJS/grapesjs#5140] [Q&A]
Hi guys, i've searched the discussions topics but could not find an answer. Is there a way to control the input precision on the above mentioned props? I use dragMode absolute and I get something like "296.46484375" from the height property of a certain component on the canvas. Is there a way to use only integers?

**Comment by artf** [comment]
There is a [fix](https://github.com/GrapesJS/grapesjs/commit/c965b35388aeda6c25297f166abdbd8d973e1de2) ready for the next release

**Reply by badiuciprian** [reply]
ok, thank you very much
